### ISSUE TITLE
Graphically display data lineage

### ISSUE THREAD

**Issue by CraigClark** [bcgov/MFIN-Data-Catalogue#312]
https://openplus.monday.com/boards/4092908516/pulses/5832336034

To better show the relationship between dependant data sets, use a flow chart to show the lineage. This should show on the node view.

![image](https://github.com/bcgov/MFIN-Data-Catalogue/assets/3910508/0c77bf8e-0061-4824-b9eb-cf6078eafd0d)

[See prototype on figma](https://www.figma.com/proto/QFQoyhpG1qvtBwVuuBVikk/design-prototypes?type=design&node-id=1195-81&t=nMzdehC2lzDZFyFS-1&scaling=min-zoom&page-id=1191%3A2738&starting-point-node-id=1195%3A81)

**Comment by CraigClark** [comment]
@lkmorlan ready for review/merge https://github.com/bcgov/MFIN-Data-Catalogue/tree/312-graphical-display-data-lineage

**Comment by CraigClark** [comment]
merged into 1.0.x

**Comment by CraigClark** [comment]
## tests

@NicoledeGreef 

Please see here https://test.cat.data.fin.gov.bc.ca/test-all-fields-filled-out-and-revisions-added and here https://test.cat.data.fin.gov.bc.ca/nicole-jan-15-updated-admin

One thing I did not test, if a data set references a MR, or is referenced by an MR I shouldn't see, does it show up here.

**Comment by CraigClark** [comment]
@NicoledeGreef I ran this through some tests and everything seems good. I think the issue was permissions on the node.

Since you are the DC admin, I have added you to all orgs.

Not sure if it's related, but Liam noted that on test we had not set flags for the public and authenticated roles. Those are now set on all thre environments and I have added a note in the documentation that this needs to be covered.
